src/pages/Header/Header.tsx

```diff
@@ -0,0 +1,67 @@
+import { useState } from 'react';
+
+import { Typography } from '@mui/material';
+
+import { HeaderContainer, LinkContainer, Title, StyledLink } from './styled';
+import TopMenu from './TopMenu';
+import { ActiveModal } from '../../types/modal';
+import AppModal from '../../components/AppModal';
+
+export const MODALS_CONTENT = {
+    Info: {
+        title: 'Information',
+        content: <p>Information modal content</p>,
+    },
+    Stats: {
+        title: 'Statistics',
+        content: <p>Statistics modal content</p>,
+    },
+    Settings: {
+        title: 'Settings',
+        content: <p>Settings modal content</p>,
+    },
+};
+
+const MENU_LINKS = Object.keys(MODALS_CONTENT) as
+    | (keyof typeof MODALS_CONTENT | null)[];
+
+const Header = () => {
+    const [activeModal, setActiveModal] = useState<ActiveModal>(null);
+
+    const onMenuChangeHandler = (newActiveModal: ActiveModal) => {
+        setActiveModal(newActiveModal);
+    };
+
+    return (
+        <HeaderContainer>
+            <Title onClick={() => onMenuChangeHandler(null)}>
+                <Typography>Redactle</Typography>
+            </Title>
+            <LinkContainer
+                sx={{ display: { sm: 'flex' } }}
+                value={activeModal}
+                exclusive
+                onChange={(_event, type) => onMenuChangeHandler(type)}
+                aria-label="header-menu"
+            >
+                {Object.keys(MODALS_CONTENT).map((menuLink) => (
```

Comment:
you can use MENU_LINKS here

Reply:
I had a problem with it here :/ but i will try to see more in to it.

Reply:
The null value was the problem, fixed it.